<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>406</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Snap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="ui_snap_Label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>391</width>
            <height>191</height>
           </rect>
          </property>
          <property name="baseSize">
           <size>
            <width>917</width>
            <height>530</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../../ui/snap.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="ui_root_Label">
          <property name="geometry">
           <rect>
            <x>105</x>
            <y>51</y>
            <width>50</width>
            <height>50</height>
           </rect>
          </property>
          <property name="baseSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string notr="true">mi</string>
          </property>
          <property name="toolTipDuration">
           <number>2</number>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
border-radius: 25px;
border-width: 2px;
border-style: solid;
border-color: rgb(20, 20, 20);
background-color: rgb(170, 170, 170);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="ui_mid_Label">
          <property name="geometry">
           <rect>
            <x>177</x>
            <y>134</y>
            <width>50</width>
            <height>50</height>
           </rect>
          </property>
          <property name="baseSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
border-radius: 25px;
border-width: 2px;
border-style: solid;
border-color: rgb(20, 20, 20);
background-color: rgb(170, 170, 170);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="ui_top_Label">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>56</y>
            <width>50</width>
            <height>50</height>
           </rect>
          </property>
          <property name="baseSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
border-radius: 25px;
border-width: 2px;
border-style: solid;
border-color: rgb(20, 20, 20);
background-color: rgb(170, 170, 170);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QTabBar::tab { height: 45px; width:209px;}</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="ui_ikTab">
          <attribute name="title">
           <string>Move IK ctrls to match target</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item row="1" column="3">
              <widget class="QPushButton" name="ui_ik_handle_Btn">
               <property name="font">
                <font>
                 <family>Bahnschrift Light</family>
                </font>
               </property>
               <property name="text">
                <string>Set IK Handle</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLineEdit" name="ui_ik_pole_LineEdit"/>
             </item>
             <item row="1" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="ui_ik_pole_Btn">
               <property name="font">
                <font>
                 <family>Bahnschrift Light</family>
                </font>
               </property>
               <property name="toolTip">
                <string>mimi</string>
               </property>
               <property name="text">
                <string>Set Pole Vector</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="2">
              <widget class="QLineEdit" name="ui_ik_handle_LineEdit"/>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="ui_fkTab">
          <attribute name="title">
           <string>Move FK ctrls to match target</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item row="1" column="5">
              <widget class="QPushButton" name="ui_fk_ctrl_top_Btn">
               <property name="font">
                <font>
                 <family>Bahnschrift Light</family>
                </font>
               </property>
               <property name="text">
                <string>Set Top</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QPushButton" name="ui_fk_ctrl_mid_Btn">
               <property name="font">
                <font>
                 <family>Bahnschrift Light</family>
                </font>
               </property>
               <property name="text">
                <string>Set Mid</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="ui_fk_ctrl_root_Btn">
               <property name="font">
                <font>
                 <family>Bahnschrift Light</family>
                </font>
               </property>
               <property name="text">
                <string>Set Root</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="2">
              <widget class="QLineEdit" name="ui_fk_ctrl_mid_LineEdit"/>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLineEdit" name="ui_fk_ctrl_root_LineEdit"/>
             </item>
             <item row="1" column="4">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="4" colspan="2">
              <widget class="QLineEdit" name="ui_fk_ctrl_top_LineEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="ui_clear_Btn">
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
            </font>
           </property>
           <property name="text">
            <string>Clear All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_snap_Btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
            </font>
           </property>
           <property name="text">
            <string>Snap</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
